### PR TITLE
Update changelog with 4.4 changes

### PR DESCRIPTION
**Pull request by ccordoba12**
@willingc, could you take a look at this one? Thanks!

### CODE REVIEW

File: docs/source/changelog.rst [hunk 1]
```diff
@@ -3,6 +3,33 @@
...
+- :kbd:`Control-D` enters an EOT character if kernel is executing and input is
+  empty.
+- Implement block indent on multiline selection with :kbd:`Tab`.
+- Change the syntax highlighting style used in the console on the fly. It can
+  be done in the menu ``View > Syntax Style``.
+
+Changes
+~~~~~~~
+- Change :kbd:`Control-Shift-A` to select cell contents first.
+- Change default tab width to 4 spaces.
```
Comment (willingc) [inline docs/source/changelog.rst]: Assuming to better accommodate Python.

File: docs/source/changelog.rst [hunk 2]
```diff
@@ -3,6 +3,33 @@
...
+.. _4.4:
+
+`4.4 on GitHub <https://github.com/jupyter/qtconsole/milestones/4.4>`__
+
+Additions
+~~~~~~~~~
+- :kbd:`Control-D` enters an EOT character if kernel is executing and input is
+  empty.
+- Implement block indent on multiline selection with :kbd:`Tab`.
+- Change the syntax highlighting style used in the console on the fly. It can
```
Comment (willingc) [inline docs/source/changelog.rst]: Perhaps instead of "on the fly" use "at any time".

File: docs/source/changelog.rst (hunk 1 above)
Comment (ccordoba12) [reply docs/source/changelog.rst]: I really don't know, I didn't do that change, sorry.

File: docs/source/changelog.rst (hunk 2 above)
Comment (ccordoba12) [reply docs/source/changelog.rst]: Ok, sounds better, thanks!

**Comment by ccordoba12** [comment]
Thanks @willingc!

**Pull request merged**